specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingSubscription.json

```diff
@@ -151,246 +237,509 @@
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/billing/"
+        },
+        "operationId": "BillingSubscriptions_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "./types.json#/parameters/billingAccountName"
+          },
+          {
+            "$ref": "./types.json#/parameters/billingSubscriptionName"
+          },
+          {
+            "$ref": "./types.json#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The billing properties of a subscription.",
+            "schema": {
+              "$ref": "#/definitions/BillingSubscription"
+            }
+          },
+          "202": {
+            "description": "An operation accepted result that returns a 'Location' header that the client can poll to view the result of the operation.",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Default response",
+            "schema": {
+              "$ref": "./types.json#/definitions/ArmErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "BillingSubscriptionsDelete": {
+            "$ref": "./examples/billingSubscriptionsDelete.json"
+          }
+        }
+      },
       "get": {
+        "tags": [
+          "BillingSubscription"
+        ],
         "description": "Gets a subscription by its ID. The operation is supported for billing accounts with agreement type Microsoft Customer Agreement and Microsoft Partner Agreement.",
         "externalDocs": {
           "url": "https://docs.microsoft.com/en-us/rest/api/billing/"
         },
         "operationId": "BillingSubscriptions_Get",
+        "produces": [
+          "application/json"
+        ],
         "parameters": [
           {
             "$ref": "./types.json#/parameters/billingAccountName"
           },
           {
-            "$ref": "./types.json#/parameters/subscriptionId"
+            "$ref": "./types.json#/parameters/billingSubscriptionName"
+          },
+          {
+            "in": "query",
+            "name": "includeDeleted",
+            "description": "Can be used to get deleted billing subscriptions.",
+            "type": "boolean",
+            "default": false
           },
           {
             "$ref": "./types.json#/parameters/apiVersionParameter"
+          },
+          {
+            "in": "query",
+            "name": "expand",
+            "description": "The expand query option specifies the related resources or media streams to be included in line with retrieved resources.",
```

Comment:
Can we describe what can be expanded? Also, should the name be "$expand", or is there precedent for using "expand"?

Reply:
@ramkd12 @amberbhargava to comment here

Reply:
Will update comment that caller can expand `Reseller` and `ConsumptionCostCenter`